### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by mohammed-sas**
### Preview
[preview link](https://deploy-preview-3--mhd-video-library.netlify.app/)
### Features
1.added video listing page with realtime data
2.added side navigation bar

PS
1.links would work in upcoming PRs
2.responsiveness will done later

### CODE REVIEW

File: src/pages/video listing/VideoListing.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import axios from 'axios'
+import VideoCard from './components/VideoCard';
+import SideNavbar from '../../components/side navbar/SideNavbar';
+const VideoListing = () => {
+    const [lists,setLists] = useState([]);
+    useEffect(()=>{
+        const fetchVideos=async ()=>{
+            try{
+                const response = await axios.get('/api/videos');
+                setLists(response.data.videos);
```
Comment (AtulPant2704) [inline src/pages/video listing/VideoListing.jsx:13]: First check `response.status`, if it is 200 then only set your list.

**Review by AtulPant2704 — APPROVED** [review]
Other then the one change mentioned, rest all looks good.

File: src/pages/video listing/VideoListing.jsx (hunk 1 above)
Comment (mohammed-sas) [reply src/pages/video listing/VideoListing.jsx:13]: sure

**Review by Kevin-Solomon — APPROVED** [review]

**Comment by Kevin-Solomon** [comment]
Clean UI. 
Great Work Mohammed.

**Pull request merged**